src/core_utils/string_core_utils.F90

```diff
@@ -0,0 +1,163 @@
+module string_core_utils
+
+    implicit none
+    private
+
+    public :: to_str           ! convert integer to left justified string
+    public :: int_date_to_yyyymmdd    ! convert encoded date integer to "yyyy-mm-dd" format
+    public :: int_seconds_to_hhmmss          ! convert integer seconds past midnight to "hh:mm:ss" format
+    public :: stringify        ! Convert one or more values of any intrinsic data types to a character string for pretty printing
```

Comment:
Total nit-pick, but can we move the comments so that they align with each other like so:

```suggestion
    public :: to_str                  ! convert integer to left justified string
    public :: int_date_to_yyyymmdd    ! convert encoded date integer to "yyyy-mm-dd" format
    public :: int_seconds_to_hhmmss   ! convert integer seconds past midnight to "hh:mm:ss" format
    public :: stringify               ! Convert one or more values of any intrinsic data types to a character string for pretty printing
```

Reply:
Fixed!